cookbooks/aws-parallelcluster-slurm/recipes/config/external_slurmdbd_disable_unrequired_services.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+#
+# Cookbook:: aws-parallelcluster-slurm
+# Recipe:: external_slurmdbd_disable_unrequired_services
+#
+# Copyright:: 2013-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Most ParallelCluster AMI are configured for Desktop use and so have some active service that is not required
+# This recipe will disable some that are certainly unused, like { apache2, cups, ...,  wpa_supplicant }
+
+# The list depends on the base OS
+# This was tested on Ubuntu 20.04, Alinux2 and Rocky8
+# Others OS should be checked and added
+
+serviceList = %w()
```

Comment:
I feel like changing this to a resource rather than a recipe would be better in the longer run.

Reply:
Though this can be an improvement  so that it doesnt block the current PR